engine/access/rpc/backend/backend.go

```diff
@@ -96,6 +98,7 @@ func New(
 			connFactory:       connFactory,
 			state:             state,
 			log:               log,
+			seenScripts:       seenScripts,
```

Comment:
Nit: no need for the extra variable.
```suggestion
			seenScripts:       make(map[[md5.Size]byte]time.Time),
```